spec/index.bs

```diff
@@ -7783,6 +7783,11 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
         1. If |sampler|.{{GPUSamplerBindingLayout/type}} is {{GPUSamplerBindingType/"filtering"}},
             then |texture|.{{GPUTextureBindingLayout/sampleType}} |must| be
             {{GPUTextureSampleType/"float"}}.
+        1. If |device|.{{device/[[features]]}} does not [=list/contain=] {{GPUFeatureName/"core-features-and-limits"}}:
+            <div class=compatmode>
+                1. If the function is `textureLoad`, |textureBinding| must not be of [=type/depth texture=] type.
+                1. If the function accepts a |samplerBinding| and |textureBinding| is of [=type/depth texture=] type, |samplerBinding| must be of `sampler_comparison` type.
```

Comment:
The parent point mentions "...together with a samplerBinding of `sampler` type (excluding `sampler_comparison`)", maybe the condition needs to be moved down in the preexisting restriction.

Reply:
LGTM, though I'm not sure about the reference question. cc @kainino0x